### ISSUE TITLE
music-on subject

### ISSUE THREAD

**Issue by blueskiy01** [01-edu/public#2442]
music-on
---
In the audit question for the 1st task "heuristic analysis" : 
Is the chart duplicated?
Is the chart complete?

However, charts was mentioned nowhere in the task.  Do you mean "evaluation checklist" instead? Deliverables are listed to be "A duplicated and completed checklist on a sheet document (Google Sheets, Airtable, Excel, etc.)."

**Comment by zanninso** [comment]
yes the charts there means evaluation checklist that might be duplicated, means that the same output should be represented in different format for different purposes such as (checklist, chart, writing report, ....)
